src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  $column-width: 200px;
+  $column-height: 300px;
 
-  max-width: 944px;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: 200px;
+  place-items: center;
+
+  justify-content: center;
   margin: 0 auto;
+  padding-bottom: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    display: grid;
+    grid-template-columns: #{$column-width};
+    grid-template-rows: #{$column-height};
```

Comment:
why do you need columns & rows in the card?
just align with center your text inside card